docs/guide/from-2023a-to-2023b.md

```diff
@@ -0,0 +1,44 @@
+# Upgrade from R2023a to R2023b
+The R2023b version of Webots introduces a new type of node: the [Pose](../reference/pose.md) node.
+It has been introduced to solve an issue regarding scale: until R2023b, solid, robots and devices had a scale parameter.
+However scaling such nodes was the cause of a lot of bugs and inconsistency.
+Moreover, in a lot of cases, it did not make sense to scale such node: you could scale down a car but motor's power where not scaled for example.
+
+This is where the Pose node comes in.
+
+## What Changes
+### What Is the Pose Node
+
+The Pose node is a descendant of [Group](../reference/group.md) and replace [Transform](../reference/transform.md) as the parent of [Solid](../reference/solid.md) and [Fluid](../reference/fluid.md).
+It has exactly the same fields than a transform but for the `scale`.
+You can see the complete node hierarchy [here](../reference/node-chart.md).
+
+### What Becomes of the Transform Node
+
+The Transform node is still there and can be used to scale `Geometry`.
```

Comment:
I would use geometries given that it is not only the Geometry node that is scaled.
And I would add `visual`, given that the bounding object geometries cannot be scaled.
```suggestion
The Transform node is still there and can be used to scale visual geometries.
```